CLI tools/MultiMC to Git.py

```diff
@@ -34,30 +36,32 @@ def copy_to_archive(from_path: Path, to_path: str, archive_path: Path) -> None:
 version = "0.0.0"
 pattern = re.compile(r"\d+\.\d+\.\d+-?\w*\.?\d*")
 
-with open(Path(mmc_path, "instance.cfg"), "r") as file:
+with open(mmc_path / "instance.cfg", "r") as file:

```

Comment:
As in the first file, need to ensure that the path won't become `...//instance.cfg`

Reply:
> As in the first file, need to ensure that the path won't become `...//instance.cfg`

Same. See my reply to your first comment.